### PR TITLE
VIVI-1342 Processing Module

### PR DESCRIPTION
**Pull request by deanylev**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      title: data.title,
+      url: data.url,
+      duration: data.duration,
+      subtitle_url: subtitleUrl,
+      cookies: data.http_headers.Cookie
+    };
+  }
+}
+
+module.exports = YtdlProcess;
```
Comment (DuBistKomisch) [inline index.js]: doesn't this mean ARGUMENTS is not longer available on module.exports?

Comment (deanylev) [reply index.js]: Nope, it's confusing but the main `require()` will be the class and `require().ARGUMENTS` will be the `ARGUMENTS`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "ytdl-process",
+  "version": "1.0.0",
+  "description": "Shared module for our youtube-dl processing",
+  "main": "index.js",
+  "dependencies": {
+    "youtube-dl": "^1.12.2"
```
Comment (DuBistKomisch) [inline package.json]: leftover

File: index.js (hunk 1 above)
Comment (DuBistKomisch) [reply index.js]:
```
jake@dubnium 0+ ~/vivi/ytdl-process vivi-1342-processing-module
 $ ls
index.js  package.json  README.md  yarn.lock

jake@dubnium 0+ ~/vivi/ytdl-process vivi-1342-processing-module
 $ node
> const ytdlProcess = require('./index');
undefined
> ytdlProcess.ARGUMENTS
undefined
```

Comment (deanylev) [reply index.js]: :facepalm: I guess I'm crazy, you have to have a `static get()` on the class then

Comment (DuBistKomisch) [reply index.js]: yeah, or just don't use a class :smile:

Comment (DuBistKomisch) [reply index.js]: OR you can export the class as a property instead of the whole export

**Review by DuBistKomisch — APPROVED** [review]
wow so short now, looks pretty good

**Pull request merged**